azurelinuxagent/ga/env.py

```diff
@@ -131,7 +132,14 @@ def _operation(self):
             self._osutil.remove_legacy_firewall_rule(dst_ip=self._protocol.get_endpoint())
             self._try_remove_legacy_firewall_rule = True
 
+        before_enable_firewall_list = self._osutil.get_firewall_list(verbose=False)
         success = self._osutil.enable_firewall(dst_ip=self._protocol.get_endpoint(), uid=os.getuid())
+        after_enable_firewall_list = self._osutil.get_firewall_list(verbose=False)
+
+        if self._log_initial_firewall_list or before_enable_firewall_list != after_enable_firewall_list:
```

Comment:
Since this is periodic operation logging very first time or if firewall list got changed after enable.